richtext_poc/templates/wagtailadmin/pages/listing/_page_title_choose.html

```diff
@@ -0,0 +1,18 @@
+{% load l10n %}
+
+{% comment %}
+The title field for a page in the page listing, when in 'choose' mode.
+
+Expects a variable 'page', the page instance.
+{% endcomment %}
+
+<h2>
+    {% if page.can_choose %}
+        <a class="choose-page" href="#{{ page.id|unlocalize }}" data-id="{{ page.id|unlocalize }}" data-slug="{{ page.slug|unlocalize }}" data-title="{{ page.get_admin_display_title }}" data-url="{{ page.url }}" data-parent-id="{{ page.get_parent.id|unlocalize }}" data-edit-url="{% url 'wagtailadmin_pages:edit' page.id %}">{{ page.get_admin_display_title }}</a>
```

Comment:
This is the exact same as the existing template, except for the `data-slug` attribute.